### DISCUSSION TITLE
CommandPalette➤Close Other Tabs does not work?

### DISCUSSION THREAD

**Discussion by nutnutwin** [sumatrapdfreader/sumatrapdf#3147] [Q&A]
Hi, thank you for your great work

# My situation
I am using pre-release v3.5.15262
where CommandPalette is implemented

# My question
- I have several tabs open
- I opened commandpalette
- I type ｢Close other tabs｣ and press Enter
- nothing happens

# Expected behavior
I was expecting tabs other than the current one is closed

Thank you.

**Comment by GitHubRulesOK** [comment]
@kjk 
also CmdCloseTabsToTheRight is non active but displayed

**Comment by kjk** [comment]
Indeed an omission. Opened https://github.com/sumatrapdfreader/sumatrapdf/issues/3262 to track this.
